core/src/main/scala/org/bitcoins/core/package.scala

```diff
@@ -31,6 +32,19 @@ package object core {
     }
   }
 
+  implicit class bigIntegerUtil(private val bigInt: BigInteger) extends AnyVal {
```

Comment:
added but not used?

Reply:
These are used in places like https://github.com/bitcoin-s/bitcoin-s/pull/2826/files#diff-f228c4c619e8243a6789f3b54ec0613dcbe13dec050796d590880a09471b9b03R25